docs/current/cookbook/snippets/cache-dependencies/index.mjs

```diff
@@ -0,0 +1,28 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(
+  async (client) => {
+    // use a node:18 container
+    // mount the source code directory on the host
+    // at /src in the container
+    // mount the cache volume to persist dependencies
+    const source = client
+      .container()
+      .from("node:18")
+      .withDirectory("/src", client.host().directory("."))
+      .withMountedCache(
+        "/src/node_modules",
+        client.cacheVolume("node-18-myapp-myenv")
+      )
+
+    // set the working directory in the container
+    // install application dependencies
+    const runner = await source
+      .withWorkdir("/src")
+      .withExec(["npm", "install"])
+      .sync()
+
+    console.log(await runner.id())
```

Comment:
Do this call to id is really useful? You already have a call to `sync` before

Reply:
Done